server/redeemer.go

```diff
@@ -160,10 +158,11 @@ type remoteSender struct {
 
 // NewRedeemerClient instantiates a new client for the ticket redemption service
 // The client implements the pm.SenderMonitor interface
-func NewRedeemerClient(uri *url.URL, sm pm.SenderManager, tm pm.TimeManager) (*RedeemerClient, error) {
+func NewRedeemerClient(uri string, sm pm.SenderManager, tm pm.TimeManager) (*RedeemerClient, error) {
 	conn, err := grpc.Dial(
-		uri.String(),
+		uri,
 		grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)),
+		grpc.WithInsecure(),
```

Comment:
Do we need this if the `tlsConfig` in the previous line already specifies `InsecureSkipVerify: true`?

Reply:
The client wouldn't connect to the server without it. 

Might also have been due to 3 sec timeout at that point (hadn't reached that one yet) , I'll double check.

You're right that we _shouldn't_ need this.

Reply:
fixed in 82eb1280